cmd/buildah/common.go

```diff
@@ -28,9 +29,10 @@ var (
 
 const (
 	maxPullPushRetries = 3
-	pullPushRetryDelay = 2 * time.Second
 )
 
+var pullPushRetriesOpts = retry.RetryOptions{MaxRetry: maxPullPushRetries}
```

Comment:
Why isn't this just retryOpts?
And above
maxPullPushRetries = 3
should be
maxRetries = 3